### PR TITLE
Routes

### PR DESCRIPTION
**Pull request by wadaudzai**
Dear reviewer,

Project Requirements:

For each URL you should create:
1: A route. 
2: An action in the correct controller.
3: A view file.
4: View files should include only placeholders written in plain HTML, e.g: Here is a list of posts for a given user
5: Make sure that you added code to handle only the 4 required URLs.

I have created a route for all the URLs.
I have created controllers with the correct actions.
I have created a view file 
All the view file have the placeholder Here is a list of posts for a given user
I make sure that it handles only the 4 required URLs.

I have worked on this project to the best of my knowledge please check it when you got time. Thanks for your time. If there is any issue with it let me know. 

Thanks

### CODE REVIEW

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    pending "add some examples (or delete) #{__FILE__}"
```
Comment (rustam-tolipov) [inline spec/requests/posts_spec.rb:5]: - [ ] In this milestone students supposed to write test for controllers. But in your project I can see only pending message. Kindly write test for your both users and post: index and show. 

> HINT:
> Spec file include tests that ensures correct view template is rendered.
> Example : expect(response).to render_template(:show)
> 
> 
>

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @wadaudzai,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by wadaudzai** [comment]
Dear @Rustamxon7 ,

I am really sorry but I accidentaly asked for a review for tomorrow's project. I asked for a review again on today's project on the same pull request if it's okay.

**Comment by wadaudzai** [comment]
Dear Reviewer,

I accidentally asked for a review for tomorrow's project. I asked for a review for today's project again. The review that I got from @Rustamxon7 is for tomorrow's project but since it has already been done. I want a review for this project. Thank you .

File: config/routes.rb [hunk 2]
```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
```
Comment (t-yanick) [inline config/routes.rb]: - [ ] Great job working on this milestone and submitting the correct project. It not professional to allow commented lines of code in our codebase. With the use of Git, we can always get a history of deleted codes. Kindly delete this line of commented code

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @wadaudzai , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Controllers generated without use of `rails generate scaffold` command.
:heavy_check_mark: Have corresponding views for generated controllers
:heavy_check_mark: Users and Posts routes added
:heavy_check_mark: Descriptive PR description and commit messages in the imperative tense

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Ensuring that you capitalize the first letter of all your commit messages

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @wadaudzai 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
- [ **OPTIONAL** ] **I think it would be better if you use [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md) for your next projects as required for all blog_app milestones**

_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**